### PR TITLE
Treeview expand animation

### PR DESCRIPTION
**Pull request by l1pton17**
It will conflict with combobox PR. You just need to keep only one version of MathMultiConverter

### CODE REVIEW

**Comment by ButchersBoy** [comment]
re. conflict, OK...let's work thru the combo box and I'll do this after.

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TreeView.xaml [hunk 1]
```diff
@@ -231,32 +225,19 @@
...
-                                                <EasingDoubleKeyFrame KeyTime="0:0:0.5" Value=".5"/>
-                                            </DoubleAnimationUsingKeyFrames>
-                                            <DoubleAnimationUsingKeyFrames Storyboard.TargetProperty="(FrameworkElement.LayoutTransform).(TransformGroup.Children)[0].(ScaleTransform.ScaleY)" Storyboard.TargetName="ItemsHost">
-                                                <EasingDoubleKeyFrame KeyTime="0" Value="1"/>
-                                                <EasingDoubleKeyFrame KeyTime="0:0:0.5" Value="0"/>
-                                            </DoubleAnimationUsingKeyFrames>
+                                            <DoubleAnimation Storyboard.TargetProperty="Opacity" Storyboard.TargetName="ItemsHost"
+                                                             To="0" Duration="0:0:0.3"/>
+                                            <DoubleAnimation Storyboard.TargetProperty="Tag" Storyboard.TargetName="ItemsScrollViewer"
+                                                             To="0" Duration="0:0:0.5"/>
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TreeView.xaml]: Expected: Duration="0:0:0.3"

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TreeView.xaml [hunk 2]
```diff
@@ -231,32 +225,19 @@
                                             <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ItemsHost" Storyboard.TargetProperty="Visibility">
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TreeView.xaml]: Expected 
```
                                         <VisualTransition.GeneratedEasingFunction>
                                             <CubicEase EasingMode="EaseOut"/>
                                         </VisualTransition.GeneratedEasingFunction>
```
 (as with above transition)

**Comment by l1pton17** [comment]
done

**Comment by ButchersBoy** [comment]
@l1pton17 did u see the other one (missing ease function)

**Comment by l1pton17** [comment]
@ButchersBoy yep

**Pull request merged**